vips/vip-304/bsctestnet.ts

```diff
@@ -0,0 +1,273 @@
+      },
+
+      // Add BabyDoge Market
+      {
+        target: BABYDOGE,
+        signature: "faucet(uint256)",
+        params: [BABYDOGE_SUPPLY],
+      },
+      {
+        target: BABYDOGE,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+      },
+      {
+        target: BABYDOGE,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, BABYDOGE_SUPPLY],
+      },
+      {
+        target: VBABYDOGE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["100"],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VBABYDOGE,
+            "300000000000000000",
+            "400000000000000000",
+            BABYDOGE_SUPPLY,
+            TREASURY,
+            "1600000000000000000000000",
+            "800000000000000000000000",
+          ],
+        ],
+      },
+
+      // Add USDT Market
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDT, USDT_SUPPLY, NORMAL_TIMELOCK],
+      },
+      {
+        target: USDT,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, 0],
+      },
+      {
+        target: USDT,
+        signature: "approve(address,uint256)",
+        params: [POOL_REGISTRY, USDT_SUPPLY],
+      },
+      {
+        target: VUSDT,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["100"],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [VUSDT, "750000000000000000", "770000000000000000", USDT_SUPPLY, TREASURY, "1000000000000", "900000000000"],
+        ],
+      },
+
+      // Add Rewards Distributor
+      {
+        target: BABYDOGE,
+        signature: "faucet(uint256)",
+        params: [REWARDS_AMOUNT],
+      },
+      {
+        target: BABYDOGE,
+        signature: "transfer(address,uint256)",
+        params: [REWARDS_DISTRIBUTOR, REWARDS_AMOUNT],
+      },
+      {
+        target: REWARDS_DISTRIBUTOR,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: COMPTROLLER,
+        signature: "addRewardsDistributor(address)",
+        params: [REWARDS_DISTRIBUTOR],
+      },
+      {
+        target: REWARDS_DISTRIBUTOR,
+        signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+        params: [[VBABYDOGE], ["12134623477230768"], ["12134623477230768"]],
+      },
+
+      // Conversions Config
+      {
+        target: RISK_FUND_CONVERTER,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [BaseAssets[0], RiskFundConverterTokenOuts, incentiveAndAccessibilityForRiskFundConverter],
```

Comment:
I think this can be simplify to:
```suggestion
        params: [BaseAssets[0], [BABYDOGE], [[0, 1]]],
```